src/symfony-bundle/composer.json

```diff
@@ -19,18 +19,18 @@
         "aeon-php/rate-limiter": "~1.0",
         "aeon-php/retry": "~1.0",
         "aeon-php/sleep": "~1.0",
-        "symfony/config": "~5.4.21||~6.3||~7.0",
-        "symfony/dependency-injection": "~5.4.21||~6.3||~7.0",
-        "symfony/form": "~5.4.21||~6.3||~7.0",
-        "symfony/http-kernel": "~5.4.21||~6.3||~7.0",
-        "symfony/validator": "~5.4.21||~6.3||~7.0"
+        "symfony/config": "^6.4 || ^7.3 || ^8.0",
```

Comment:
is dropping 5.4 necessary? If it's not creating any additional maintenance work, it should stay. Supported version of dependencies should stay as wide as possible